config/prod/ahayden-testhost-win.yaml

```diff
@@ -0,0 +1,50 @@
+# Provision a Windows EC2 instance
+template_path: remote/managed-ec2-win-v4.j2
+stack_name: ahayden-testhost-win
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "Platform"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "Infrastructure"
+  # The resource owner
+  OwnerEmail: "aaron.hayden@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "computevpc"
+  KeyName: "scicomp"
+  AMIId: "ami-0388ce15d3f7066cb"    # Windows_Server-2019-English-Full-2019.03.13
```

Comment:
this is not the same AMI as in the example template.  https://github.com/Sage-Bionetworks/scicomp-provisioner/pull/128/files#diff-f5e07fc328d4bdc4c00d094873166699R15

if you want this AMI could you please add a comment with info about it?

Reply:
It's the default Win AMI in config.yaml?... Remember the switcharoo with AMIs? Could it be the example template does not have the AMI ID that AWS lists after the switcharoo?

Reply:
i have updated the example template.  https://imgur.com/a/ANVefVU